js/components/directionControl.js

```diff
@@ -0,0 +1,39 @@
+var React = require('react');
+var classnames = require('classnames');
+
+var DirectionControl = React.createClass({
+  getInitialState: function() {
+    return {
+      isTouched: false,
+    };
+  },
+
+  handleEvent: function (ev) {
+    var rotate = ev.type == 'mousedown' || ev.type == 'touchstart';
+    this.props.handleDirection(rotate, this.props.dir);
+    
+    this.setState({
+      isTouched: rotate
+    });
+  },
+
+  render: function () {
+    var directionClass = 'oo-vr-icon--move--' + this.props.dir
+      , touchedDirectionClass = this.state.isTouched && 'oo-vr-icon--move--' + this.props.dir + '--touched';
```

Comment:
This will evaluate to false if isTouched is false. Please take a look at this.

Reply:
Thank you, Alexander.
But touchedDirectionClass is used in classnames, so if touchedDirectionClass is false, classnames ignores it.